src/components/GiftInput.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Container = styled.div`
+    padding: 30px;
+`
+
+const Input = styled.input`
+    width: 300px;
+    border-radius: 5px;
+    padding: 10px;
+    border: 2px solid #AC47D3;
+    font-size: 16px;
+    margin-right: 30px;
```

Comment:
Maybe you could make the inputs and the submit button the same height? You could add `height: 50px` and `box-sizing: border-box;` to the input, and `height: 50px;` to the submit button
🚚